src/test/java/com/wikia/webdriver/common/driverprovider/mobileprovider/MobileDriverProvider.java

```diff
@@ -21,23 +22,19 @@ public MobileDriverProvider() {
   public WebDriver getDriverInstance() {
     WebDriver driver = null;
 
-    if ("CHROMEMOBILEMERCURY".equals(browser)) {
+    if (Browser.CHROME_MOBILE_MERCURY.equals(browser)) {
       return NewDriverProvider.getDriverInstanceForBrowser(browser);
     }
 
     switch (platform.toUpperCase()) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Replace this "switch" statement by "if" statements to increase readability. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1301)